### PR TITLE
Make Garrick maintainer. Add Alex + Nischal as `ctb`

### PR DESCRIPTION
**Pull request by schloerke**
cc @gadenbuie @nischalshrestha @rossellhayes

### CODE REVIEW

**Comment by coatless** [comment]
@schloerke to be fair, @rossellhayes probably desires an author credit based on Git commit history. 

Some other authors probably could be bumped down to ctb.

**Comment by gadenbuie** [comment]
Thanks @coatless, but the `aut` role is a reflection of the team responsible for the development, maintenance and direction of the package, rather than the volume of commits. For the time being `ctb` is appropriate, and if (or hopefully when!) @rossellhayes's role with learnr expands in the future we'll update the authors list accordingly.

File: DESCRIPTION [hunk 1]
```diff
@@ -3,24 +3,34 @@ Package: learnr
...
              comment = c(ORCID = "0000-0001-9986-114X")),
       person(given = "JJ",
              family = "Allaire",
-             role = "aut",
+             role = c("aut", "ccp"),
              email = "jj@rstudio.com"),
       person(given = "Barbara",
              family = "Borges",
              role = "aut",
              email = "barbara@rstudio.com"),
```
Comment (schloerke) [inline DESCRIPTION:23]: Remove Barbara

```suggestion
```

File: DESCRIPTION [hunk 2]
```diff
@@ -3,24 +3,34 @@ Package: learnr
...
+      person(given = "Nischal",
+             family = "Shrestha",
+             role = "ctb",
+             email = "nischal@rstudio.com",
+             comment = c(ORCID = "0000-0003-3321-1712")),
+      person(given = "Alexander",
+             family = "Rossell Hayes",
+             role = "ctb",
+             email = "alex.rossellhayes@rstudio.com",
+             comment = c(ORCID = "0000-0001-9412-0457")),
```
Comment (schloerke) [inline DESCRIPTION:33]:
```suggestion
      person(given = "Alexander",
             family = "Rossell Hayes",
             role = "ctb",
             email = "alex.rossellhayes@rstudio.com",
             comment = c(ORCID = "0000-0001-9412-0457")),
      person(given = "Nischal",
             family = "Shrestha",
             role = "ctb",
             email = "nischal@rstudio.com",
             comment = c(ORCID = "0000-0003-3321-1712")),
```

**Review by gadenbuie — APPROVED** [review]

**Pull request merged**